christine-anthony.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<html lang="en">
+<html>
+
+  <head>
+    <meta charset="uft-8">
+  <!-- why doesnt the fond change??? -->
+    <title>Of Patterns and Power: Web Standards Then & Now </title>
+      <meta name="description" content="HTML article titled Of Patterns and Power: Web Standards Then and Now with many links to other articles.">
+
+
+  </head>
+
+  <body>
+    <header>
+      <img source="images/exe1-1.jpeg" alt="HTML5 Doctor Logo"WIDTH=32 HEIGHT=32>
+    <!-- why doesnt it know where to get the img??? -->
```

Comment:
Do  you have an image by that file name in your `images` directory?

Reply:
its in the images directory in the html exercise-1 that i forked/cloned for this project. wouldn't that work? I'll try out a few other things...